protocol/common/cobra_common.go

```diff
@@ -32,6 +32,8 @@ const (
 	// Disable relay retries when we get node errors.
 	// This feature is suppose to help with successful relays in some chains that return node errors on rare race conditions on the serviced chains.
 	DisableRetryOnNodeErrorsFlag = "disable-retry-on-node-error"
```

Comment:
we don't need both flags (set retries and disable, combine them to one flag, if it's 0 do disable